task/sast-shell-check-oci-ta/0.2/sast-shell-check-oci-ta.yaml

```diff
@@ -0,0 +1,259 @@
+---
```

Comment:
Why are we introducing `sast-shell-check-oci-ta/0.2` when there is no `sast-shell-check/0.2`?

Reply:
by mistake, 
agree with you, the parameter already exist so I only add it within the oras uploading command